### PR TITLE
BETA: Register zymi.is-a.dev

### PR DESCRIPTION
**Pull request by Zyminotfound**
Added `zymi.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by triach-rold — APPROVED** [review]

**Comment by triach-rold** [comment]
Oops. I accidentally approved the wrong PR. mb, had multiple PRs open on my monitor.

File: domains/Zymi.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/Zyminotfound/Zyminotfound.github.io",
+    "owner": {
+        "username": "Zyminotfound",
+        "email": "notzymi0@gmail.com",
```
Comment (CuteDog5695) [inline domains/Zymi.json:6]: remove the comma at the end in email line

File: domains/Zymi.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    },
+    "record": {
+        "CNAME": "Zymi.github.io"
```
Comment (CuteDog5695) [inline domains/Zymi.json:9]: 404 page

File: domains/zymi.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Zyminotfound",
+        "email": "notzymi0@gmail.com"
+    },
+    "record": {
+        "CNAME": "zymi.github.io"
```
Comment (CuteDog5695) [inline domains/zymi.json:7]: 404 page

File: domains/Zymi.json
Comment (CuteDog5695) [inline domains/Zymi.json:1]: all letters need to be lower case here, but since you have another json file thats almost the same name delete one of them

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/zymi.json (hunk 3 above)
Comment (Zyminotfound) [reply domains/zymi.json:7]: how do i fix it?

File: domains/zymi.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Zyminotfound",
+        "email": "notzymi0@gmail.com"
+    },
+    "record": {
+        "CNAME": "zyminotfound.github.io"
```
Comment (DEV-DIBSTER) [inline domains/zymi.json:7]: 404 GitHub page.

**Review by DEV-DIBSTER — DISMISSED** [review]

**Review by iostpa — APPROVED** [review]
its fine now, website is not hosted directly on zyminotfound.github.io so thats why it gives a 404 error

**Pull request merged**